README.md

```diff
@@ -1,3 +1,11 @@
 # Heroku Incidents
 
+
+
+## Incident Timeline
+
+![Heroku Incidents Timeline](incidents.svg)
```

Comment:
Does GitHub allow embedding SVGs like this? Will it render? In the past I’ve had to append `?sanitize=true` but maybe that’s not necessary any more.